automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/monitor/SubmissionDetailsPage.scala

```diff
@@ -80,7 +80,10 @@ class SubmissionDetailsPage(namespace: String, name: String, var submissionId: S
     */
   def waitUntilSubmissionCompletes(timeOut: FiniteDuration = 20.minutes): Unit = {
     retry[Boolean](5.seconds, timeOut) ({
-      goToMonitorTab().openSubmission(submissionId)
+      if (workflowStatusLabel.isVisible) {
```

Comment:
clicking tab `Monitor` switches page between `Workflow` and `Submission`. `submissionId` is on `Submission` page, it's not on `Workflow` page.